pkg/network/driver_interface.go

```diff
@@ -58,10 +58,10 @@ func makeDDAPIVersionBuffer(signature uint64) []byte {
 type DriverInterface struct {
 	driverFlowHandle      *DriverHandle
 	driverStatsHandle     *DriverHandle
-	enableMonotonicCounts bool
 
 	path       string
 	totalFlows int64
```

Comment:
the idea is you want totalFlows first, since that's what's passed to the `atomic` functions.

Reply:
What do you mean by "first"? Does moving the declaration to after totalFlows not achieve this, or do I need to have totalFlows be the first thing declared in the struct?

Reply:
I am not sure how `string` would be aligned. Safer to move it after the `*DriverHandle` pointers.

Reply:
Thanks for the note - put it at the top with a comment!